deploy/production/ingress.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: justice-gov-uk-production-ingress
+  namespace: justice-gov-uk-production
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: justice-gov-uk-production-ingress-justice-gov-uk-production-green
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/auth-type: basic
+    nginx.ingress.kubernetes.io/auth-secret: basic-auth-secret
```

Comment:
We can remove basic auth, only we should reach it over host file redirect

Reply:
Would be good to have a chat about this.

I don't think we factored in that we need to have Antony switch over the DNS, **then** get an ssl cert.

We could get around it...

- Create an a record to squiz. 
- Antony switches the DNS to Cloud Platform. 
- Squiz content is still being served.
- Add Certificate.yaml to cloud-platform-environments